src/push/monitor.ts

```diff
@@ -268,6 +268,7 @@ export function buildMonitorFromYaml(
     enabled: config.enabled ?? options.enabled,
     locations: options.locations,
     tags: options.tags,
+    labels: parseLabels(config, options.labels),
```

Comment:
Please add this new field to some of the unit tests validating lightweight monitors

Reply:
i have added here [`200e52f` (#959)](https://github.com/elastic/synthetics/pull/959/commits/200e52f7622ad28985d46c42c4723399ddd54533)